### ISSUE TITLE
(-x) exclude regions based on given BED file

### ISSUE THREAD

**Issue by gpertea** [gpertea/stringtie#24]
It would be good to be able to extend the -x option to not just exclude whole chromosomes/contigs from stringtie processing, but also genome regions, and using a BED file would be a natural way to provide that list of regions to exclude.

**Comment by sung** [comment]
Yes please. It would be nice to have this sort of 'masking' option equivalent -M/–mask-file of Cufflinks.

**Comment by meono** [comment]
I've been looking for masking feature myself. Would be great to have it.

**Comment by wangyugui** [comment]
This is an important feature to mask rRNA in all samples.

**Comment by zoewardnz** [comment]
Hi,
Any news as to when this feature will be implemented??? It would be extremely useful.
